addons/sale_project/models/project.py

```diff
@@ -387,19 +417,18 @@ def _get_revenues_items_from_sol(self, domain=None, with_action=True):
                     .browse(other_revenues.pop('record_ids', [])) \
                     ._filter_access_rules_python('read')
                 if sale_order_items:
-                    if sale_order_items:
-                        args = [section_name, [('id', 'in', sale_order_items.ids)]]
-                        if len(sale_order_items) == 1:
-                            args.append(sale_order_items.id)
-                        action_params = {
-                            'name': 'action_profitability_items',
-                            'type': 'object',
-                            'args': json.dumps(args),
-                        }
-                        other_revenues['action'] = action_params
-        sequence_per_invoice_type = self._get_profitability_sequence_per_invoice_type()
+                    args = [section_name, [('id', 'in', sale_order_items.ids)]]
+                    if len(sale_order_items) == 1:
+                        args.append(sale_order_items.id)
+                    action_params = {
+                        'name': 'action_profitability_items',
+                        'type': 'object',
+                        'args': json.dumps(args),
+                    }
+                    other_revenues['action'] = action_params
+        data += [{'id': invoice_type, 'sequence': sequence_per_invoice_type[invoice_type], **vals} for invoice_type, vals in revenues_dict.items()]
```

Comment:
```suggestion
        data += [{
            'id': invoice_type,
            'sequence': sequence_per_invoice_type[invoice_type],
            **vals
         } for invoice_type, vals in revenues_dict.items()]
```
Tant qu'on y est :angel: